coding-standards/html-css/README.md

```diff
@@ -1,40 +1,93 @@
+    display: block; width: 100%; height: 100%;
+    background: $blue; color: $white;
+}
+```
+
+If we would just append properties on the fly, we would always be looking for them, resulting in sloppy code and possible regressions since you might miss important properties:
+
+```css
+.class {
+    width: 100%; height: 100%;
+    background: $blue;
+    color: $white;
+    position: absolute; top: 0;
+    display: block; z-index: 100; left: 0;
+}
+```
+
+By grouping and ordering properties in a consistent manner, we can see at a glance what the element will look like and how it's positioned. This reduces a lot of cognitive load.
+
+
+### Utility classes
+
+Avoid using utility classes, especially for overall styling. It will inevitably lead to unforseen bugs and unnecessary complexity.
+
+- They're usally combined, resulting in specificity issues. The order of the property does not matter, only the cascade in our stylesheet dictates which property 'wins'.
+- It creates unnecessary cognitive load, since you might have to combine the properties of 5 classes in your head to get the complete 'selector'.
+- Some developers might be more familiar with them than others. This easilly leads to bugs, since changing the styling for element A, also changes the styling for element B, F and Y.
+- You might want to have a small variation since element C needs to look slightly different than element B, leading to possibly many variations of a simple utility class.
+- It creates the false illusion that a back-end developer can create the styling for a newly created feature by glueing utility classes toghether. In general, it's usally better to let a dedicated front-end developer be the final judge.
+
+Can you tell what the following element will look like?
+
+```html
+<div class="text block widget widget--small margin-top left"></div>
+```
+
+You might have a general idea. But you still would need to look up all those classes, and combine them in your head to get the full picture. Furthermore, changing `.text` for this specific element, might break another element using that same `.text`.
+
+```html
+<div class="specific-module"></div>
+```
+
+Using a specific module class for our element, means that all our styling is scoped to that module. This means that we can look up `.specific-module` and be sure that everything is in there. Changing that module will only change (and possibly break) that module.
+
+Note: using scoped modules means that we sometimes might have duplicate groups of properties, which are shared between multiple modules. This however, does not wheigh up to the fact that it's a lot safer to work with scoped modules. Also variation and exceptions are fairly common, which means that only a subset is usally shared between modules. Furthermore, you can always write a simple mixin and import that in your module. Beware that this can lead to generic 'utility'-mixins as well, and only shifts the cause of the problem elsewhere.
```

Comment:
Adding a comment would be another option. Of course not the first and the comment must be short, but it can shine soms light on code/changes.

Reply:
You mean adding a comment/warning to a utility class (if not, could you clarify)? That could help, although most of the arguments still hold up. And if we can prevent having to add comments by making (almost) everything scoped, then I'd vote for that. 

PS: we still use utility/helper stuff, but that's mostly in the form of really simple mixins nowadays. Think of `.module__button { @include hide-text; }`, instead of `class="module_button hide-text"`. This means that all the styling-related stuff is at least bundled together that styling-module.

Reply:
My comment is an addition to "Note: ". I don't want to remove any of the content you wrote. Adding comments could be useful when code is duplicated and creating a utility classes is not desirable. When two modules look almost the same, it could be useful to point developers to the differences. Prevents developers from searching for differences and it's more easy to find common peaces of code.

```
// Looks like blog but with a different color scheme.
.module-news {
    // A list of properties
}

// Looks like news but with a different color scheme.
.module-blog {
    // The same list of properties from .module-news, except:
    color: green;
}
```